src/redux/configureStore.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+import Reducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const store = configureStore({
+  reducer: { books: Reducer, categories: categoriesReducer },
```

Comment:
- [ ] for applying best practices , kindly add a meaningful name to your book reducer like you did for the categories reducer , you can use something like `booksReducer` instead of just `Reducer` , kindly fix it 🙏

Reply:
Thanks for the review.